Buildings/Applications/DataCenters/ChillerCooled/Examples/BaseClasses/PostProcess.mo

```diff
@@ -0,0 +1,126 @@
+within Buildings.Applications.DataCenters.ChillerCooled.Examples.BaseClasses;
+partial model PostProcess "Post-processing"
+
+  Buildings.Utilities.Math.IntegratorWithReset FCHou(k=1/3600)
```

Comment:
I suggest to do all the unit conversion in python script that post-process the results. Although you mention the unit in the name, I think there is a risk people will make unit errors. This would also be the first model in the whole library that does such unit conversion.